modules/claims_api/spec/lib/claims_api/special_issue_mappers/bgs_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'claims_api/special_issue_mappers/bgs'
+
+describe ClaimsApi::SpecialIssueMappers::Bgs do
+  [
+    { name: 'Character of Discharge', code: 'CD' },
+    { name: 'PTSD/1', code: 'PTSD/1' }
+  ].each do |special_issue|
+    it "returns correct code for name: #{special_issue[:name]}" do
+      expect(subject.code_from_name!(special_issue[:name])).to eq(special_issue[:code])
+    end
+  end
+
+  it 'returns nil for invalid name' do
```

Comment:
Could have a test for the ! version to make sure it raises.  (but ya don't have to 😉)

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/6013/files#diff-f667abb030578efac5a8fcc97cb99d00d7fb418afda7a04065f97db46470b291R20

and

https://github.com/department-of-veterans-affairs/vets-api/pull/6014/files#diff-b4fd875aa5bb80c833d125d38929e9fb6528fc260574645381dd4fd811d59d30R20